cmd/topology-aware/policy/topology-aware-policy.go

```diff
@@ -189,15 +190,15 @@ func (p *policy) Rebalance() (bool, error) {
 
 	for _, c := range movable {
 		if err := p.AllocateResources(c); err != nil {
-			if errors == nil {
-				errors = err
+			if errs == nil {
+				errs = err
 			} else {
-				errors = policyError("%v, %v", errors, err)
+				errs = policyError("%v, %v", errs, err)
```

Comment:
Wouldn't it be better to collect the errors here with something like `errs = errors.Join(errs, policyError("%v", err))` (regardless of what the original code did) ?

Reply:
yes, I missed that part.